src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -197,6 +197,27 @@ def test_pyttsx(pyi_builder):
         """)
 
 
+@importorskip('pyttsx3')
+def test_pyttsx3(pyi_builder):
+    import os
+
+    SOURCE = """
+        import os
+        import pyttsx3
+        engine = pyttsx3.init()
+    """
+    # The above is enough to cause a failure without the hook. It's a lot more
+    # reassuring if it includes the block below too.
+    # But on Windows CI this blocks indefinitely so exclude it in that case.
```

Comment:
I'm not sure if having the test actually speak out the sentence is a good idea. It does ensure that everything works, but does it gracefully handle the case when there's no audio output device present on the system? On the flipside, having it suddenly speak out when running tests on your local PC might also be unexpected...

Reply:
Ok it's gone.